src/auxkernels/MFEMCurlAux.C

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include "MFEMCurlAux.h"
+
+registerMooseObject("PlatypusApp", MFEMCurlAux);
+
+/*
+Class to set an H(div) auxvariable to be the curl of a H(curl) vector variable.
+*/
+InputParameters
+MFEMCurlAux::validParams()
+{
+  InputParameters params = MFEMAuxKernel::validParams();
+  params.addClassDescription(
+      "Calculates the curl of a H(curl) conforming ND source variable and stores the result"
+      " on a H(div) conforming RT result auxvariable");
+  params.addRequiredParam<VariableName>("source",
+                                        "Vector H(curl) MFEMVariable to take the curl of.");
+  params.addParam<mfem::real_t>("scale_factor", 1.0, "Factor to scale result auxvariable by.");
+  return params;
+}
+
+MFEMCurlAux::MFEMCurlAux(const InputParameters & parameters)
+  : MFEMAuxKernel(parameters),
+    _source_var_name(getParam<VariableName>("source")),
+    _source_var(*getMFEMProblem().getProblemData()._gridfunctions.Get(_source_var_name)),
+    _scale_factor(getParam<mfem::real_t>("scale_factor")),
+    _hcurl_fespace(*_source_var.ParFESpace()),
+    _hdiv_fespace(*_result_var.ParFESpace()),
```

Comment:
Do these actually need to be stored as class members? Can't you just access them to construct the curl operator, without saving references to them?